datanode/src/slippy_util.py

```diff
@@ -41,6 +41,8 @@
 logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
 log = logging.getLogger('InterUSS_DataNode_SlippyUtil')
 
+TILE_LIMIT = 50  # Number of tiles to allow for multi get/put/del
```

Comment:
Do we foresee a use case for this many tiles?  If operations can only reasonably be as big as a single tile, then we'd want this limit to be 4.  If they might be slightly larger than a tile, then 9 might be appropriate.  50 seems very large.

Reply:
For the main platform, zoom level 14 or 15 is expected. That gives a width of < 1km. 45 degree angle of a 10km flight might be near 40? Performance-wise, 50 seems manageable in web timeouts, since it is committed as a transaction.